landing-page/src/reportWebVitals.js

```diff
@@ -0,0 +1,13 @@
+const reportWebVitals = onPerfEntry => {
```

Comment:
do you know what these web vitals are? I know they are generated by react, but are they useful?

Reply:
yeah they are generated by react, I don't know if they are useful or not

Reply:
might as well toss em if they ain't useful. Less clutter.

Reply:
Can probably just toss this whole file `reportWebVitals.js`, unless there's a reason for it?

Reply:
I tried, but then I was getting a bunch of errors. I trimmed it to what looks like the minimum needed to keep react happy